camel/storages/vectordb_storages/qdrant.py

```diff
@@ -0,0 +1,350 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from dataclasses import asdict
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+from camel.storages.vectordb_storages import (
+    BaseVectorStorage,
+    VectorDBQuery,
+    VectorDBQueryResult,
+    VectorDBStatus,
+    VectorRecord,
+)
+from camel.types import VectorDistance
+
+_qdrant_local_client_map: Dict[str, Tuple[Any, int]] = {}
+
+
+class QdrantStorage(BaseVectorStorage):
+    r"""An implementation of the `BaseVectorStorage` for interacting with
+    Qdrant, a vector search engine.
+
+    The detailed information about Qdrant is available at https://qdrant.tech/.
+
+    Args:
+        vector_dim (int): The dimenstion of storing vectors.
+        collection (Optional[str], optional): Name for the collection in the
```

Comment:
Here `collection` is used as the name of the collection, but underneath there is some use of `collection_name` as the variable name in other functions. While this doesn't cause a bug, I think it leads me to misunderstand that `collection` is more than just the collection name. What do you think about changing the variable name here to `collection_name`?

Reply:
I agree it makes sense to call it `collection_name` as well.